File: reverse_and_digitize_number/better.rb
def digitize(n)
    # convert the number to a string
    # reverse the string
    # split the string into an array
    split_str = n.to_s.reverse.split("")

    # declare an empty array variable
    # this array will contain our digitize integers
    int_arr = []
    
    # iterate through the split string
    # convert each character to an integer
    # push that integer into the array
    split_str.each {|i| int_arr << i.to_i}
    
    # implicit return
    int_arr
end

File: string_manipulation/reverse_and_digitize_number/better.py
def digitize(n):
    # declare an empty list variable
    # this is where we will add the digitized numbers
    split_list = []

    # convert the number to a string
    # iterate through the string
    for c in str(n):
        # convert each character into an integer
        # append the character at index 0
        split_list.insert(0, int(c))
        
    # return the list
    return split_list